### PR TITLE
Image Question and Drawing Tool updates

### PR DESCRIPTION
**Pull request by pjanik**
1. Image Question saves PNG when the user closes the dialog using "Close" button
2. Image Question and Drawing Tool layout is updated to match LARA styles
3. Image Question uses special metadata types that will let LARA and ActivityPlayer export it in the same way as the classic LARA image question. It's necessary so the answer image URL can be displayed in the reports.

### CODE REVIEW

File: src/image-question/components/runtime.tsx [hunk 1]
```diff
@@ -18,31 +20,70 @@ const drawingToolDialogUrlParam = "drawingToolDialog";
...
     setControlsHidden(false);
-    showModal({ type: "dialog", url: window.location.href + "?" + drawingToolDialogUrlParam });
+    if (success) {
+      openDrawingToolDialog();
+    }
+  };
+
+  const openDrawingToolDialog = () => {
+    // notCloseable: true disabled click-to-close backdrop and X icon in the corner. Dialog can be closed only
+    // manually by calling closeModal() from one of our own widgets.
```
Comment (kswenson) [inline src/image-question/components/runtime.tsx:39]: "manually" suggests user intervention; how about "via API" instead?

File: src/shared/hooks/use-property-update.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import usePrevious from "react-hooks-use-previous";
+import { useState } from "react";
+
+const loading = "loading";
+
+// A simple hook that accepts object and a property name and returns `true` when the property has been updated.
+// There's an assumption that the object can be initially unavailable (undefined) and this should not affect
+// the result. This hook will detect change of the property, not property and object combination.
+// It's useful for observing interactive state properties.
+export const usePropertyUpdate = (object: any, propertyName: string) => {
```
Comment (kswenson) [inline src/shared/hooks/use-property-update.ts:10]: I found the name `usePropertyUpdate` less than transparent on first reading. How would you feel about `usePropertyDidChange` or some variant thereof?

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Looks good -- just a few minor suggestions.

File: src/shared/hooks/use-property-update.ts (hunk 2 above)
Comment (pjanik) [reply src/shared/hooks/use-property-update.ts]: I renamed it, thanks. All the naming suggestions are welcome. ;)

**Review by kswenson — APPROVED** [review]
👍 Changes look good!

**Pull request merged**